lib/environment.js

```diff
@@ -32,6 +35,9 @@ exports.Environment = Environment = function(options) {
    delete options.tags;
    delete options.debug;
    this.config = options;
+   
+   this.resourcesDir = this.loader && typeof(this.loader.findResourcesDir) === "function" ? this.loader.findResourcesDir() : "./resources";
```

Comment:
I don't understand this: The filesystem Loader looks for a given file in multiple base paths and returns the first match. It's useful for overriding base templates with more specialized variations. ("mysite/base", "mysite/customer1", ...) I don't know if this is helpful for i18n resources.